tests/dragon/utils/msg_pump.py

```diff
@@ -174,6 +174,8 @@ def _mock_messages(
     # keep the process alive for an extra 15 seconds to let the processor
     # have access to the channels before they're destroyed
     for _ in range(15):
+        # TODO: Chris says he wants to fix something here!
```

Comment:
Mr Memory over here doesn't remember either. Feel free to remove this comment and I'll create a ticket when i remember..

Reply:
removed!